brainrender_napari/widgets/atlas_manager_view.py

```diff
@@ -74,51 +74,42 @@ def _on_row_double_clicked(self):
             )
             download_dialog.exec()
 
-    def _start_worker(
-        self,
-        operation: Callable,
-        atlas_name: str,
-        signal: Signal,
-        operation_type: str,
-    ):
-        """
-        Helper function that connects the `progress_updated` signal to
-        the underlying atlas API progress update function, and then starts
-        a download/update operation in separate thread,
-        ensuring it returns `signal` when the thread operation finishes.
-        """
+    def _on_download_atlas_confirmed(self):
+        """Downloads the currently selected atlas and signals this."""
+        atlas_name = self.selected_atlas_name()
 
-        def update_fn(completed, total):
-            self.progress_updated.emit(
-                completed, total, atlas_name, operation_type
-            )
+        # Define update function directly
+        update_fn = lambda completed, total: self.progress_updated.emit(
+            completed, total, atlas_name, "Downloading"
+        )
 
+        # Create worker, connect signal, and start worker
```

Comment:
```suggestion
```
Some advice on good practice: comments that explain the _intent_ of the code are useful (e.g. _why_ we are doing something), comments that repeat _what_ the code does are not so useful (apart from in docstrings!).

I would remove the comments you added in this PR.

You can read more in-depth thinking about this[ in this blog.](https://chrlschn.medium.com/on-code-comments-49e40b2ec27e)